parsec/core/backend_connection/transport.py

```diff
@@ -52,6 +62,7 @@ async def apiv1_connect(
         BackendConnectionError
     """
     handshake = APIV1_AnonymousClientHandshake(addr.organization_id)
+    assert addr.port is not None, "Organization port is None"
```

Comment:
I think this is an error in the `pyi`, I'd expect `addr.port` to never be `None` (from what I see in the bindings)

Reply:
[The underlying rust struct](https://github.com/Scille/parsec-cloud/blob/7442bb3106107aa6314518cf010ca9dfb7d45261/oxidation/libparsec/crates/types/src/addr.rs#L157) takes an `Option<u16>` as port. Is there a default port number?